### PR TITLE
Pin github actions and add dependabot actions ecosystem

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (ludeeus) [inline .github/dependabot.yml:6]: Is daily really needed?

```suggestion
      interval: weekly
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+    open-pull-requests-limit: 10
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (ludeeus) [inline .github/dependabot.yml:11]: Is daily really needed?

```suggestion
      interval: weekly
```

**Review by ludeeus — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (balloob) [reply .github/dependabot.yml:6]: Yeah let's not do daily. It spams so much 🙃

Comment (MartinHjelmare) [reply .github/dependabot.yml:6]: I think it's nice to not have it on the same day as the pip bumps.

Comment (MartinHjelmare) [reply .github/dependabot.yml:6]: And the action bumps for the actions we normally use don't spam very much in my experience.

**Review by frenck — APPROVED** [review]

**Pull request merged**